azurerm/resource_arm_virtual_machine_scale_set_test.go

```diff
@@ -301,8 +301,8 @@ func TestAccAzureRMVirtualMachineScaleSet_planManagedDisk(t *testing.T) {
 func TestAccAzureRMVirtualMachineScaleSet_customImage(t *testing.T) {
 	ri := acctest.RandInt()
 	resourceGroup := fmt.Sprintf("acctestRG-%d", ri)
-	userName := "testadmin"
-	password := "Password1234!"
+	userName := fmt.Sprintf("testadmin%d", ri)
+	password := fmt.Sprintf("Password1234!", ri)
```

Comment:
Is this gaining the desired effect? Just noticed there's no verbs in this format string.